modules/ROOT/pages/mutations/create.adoc

```diff
@@ -139,22 +154,13 @@ mutation {
 
 This will ensure that a movie with ID 1234 exists and it is connected to `"Tom Hanks"`. If the movie does not exist, it will be created with the title `"Forrest Gump"`. Note that if the movie with the given ID already exists, it will be connected to it, regardless of the title.
 
-[[optimizing-create-operations]]
-== Optimizing create operations
-
-It's possible to use the Neo4jGraphQL library to create several nodes and relationships in a single mutation, however,
-it's well known that performance issues are present in performing this task.
-A solution has been implemented that doesn't require any changes from the user.
-However, there are still several situations where this kind of optimization is not achievable.
-
-=== Subscriptions enabled
-
-No optimizations are available if a Subscription plugin it's being used.
-
-=== `@populated_by`
+== Batch `create` optimization
```

Comment:
I wouldn't mention "batch" here - I don't know why we ever started using that word but I don't really think it's applicable.

```suggestion
== Create optimization
```